canonical/ubuntu.com
# Add redirect from snapd-whitepaper engage page to core page

## Done

- Add redirect from /engage/snapd-whitepaper engage page to /core/features/ota-updates

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/engage/snapd-whitepaper
- Should redirect to  http://0.0.0.0:8001/core/features/ota-updates
